Sources/SwiftUI Apple Watch Decimal Pad/DigetPadView.swift

```diff
@@ -160,15 +169,15 @@ public struct EnteredText: View {
 			HStack(spacing:widthSpace) {
                 if style == .decimal {
                     Button(action: {
-                        if !(text.contains(".")){
+                        if !(text.contains(decimalSeparator)){
                             if text == ""{
-                                text.append("0.")
+                                text.append("0\(decimalSeparator)")
                             }else{
-                                text.append(".")
+                                text.append(decimalSeparator)
                             }
                         }
                     }) {
-                        Text("•")
+                        Text(decimalSeparator)
```

Comment:
Will the separator be cut off on the watch, since it is lower now?

Reply:
Looks like this now 
|<img width="378" alt="Screenshot 2021-12-16 at 10 29 38" src="https://user-images.githubusercontent.com/22814132/146345070-0a5e72bc-589a-4f9a-910c-b0db9d4a6c33.png">|<img width="378" alt="Screenshot 2021-12-16 at 10 31 47" src="https://user-images.githubusercontent.com/22814132/146345396-9e37af24-aa85-4713-8690-3b04c7fa0f69.png">|
|-|-|

Reply:
Ok, good enough. We can think about adding padding in a separate iteration. Please open a PR in the main repo after.